SlicerConfiguration/Settings/PrinterSettings.cs

```diff
@@ -371,6 +371,23 @@ public string GetValue(string sliceSetting, IEnumerable<PrinterSettingsLayer> la
 			return "";
 		}
 
+		public bool Contains(string sliceSetting, IEnumerable<PrinterSettingsLayer> layerCascade = null)
+		{
+			if (layerCascade == null)
+			{
+				layerCascade = defaultLayerCascade;
+			}
+			foreach (PrinterSettingsLayer layer in layerCascade)
+			{
+				string value;
+				if (layer.TryGetValue(sliceSetting, out value))
```

Comment:
Consider layer.ContainsKey here since the unused variable and assignment could be skipped and it might better convey intent